projects/pastanaga/src/lib/button/button.component.scss

```diff
@@ -18,45 +18,35 @@ pa-button {
     vertical-align: middle;
     border-radius: 10rem;
     font-family: $font-family-base;
-    text-transform: uppercase;
+    text-transform: $labels-text-transform;
```

Comment:
might be a bit too generic: are we sure all the labels through pastanaga will have the same text transform?